index.js

```diff
@@ -51,15 +51,131 @@ what's on disk?
     /story-2
 */        
 
+// Generate image using a text prompt, and save it within the directoryPath. The file path name should include the index
+// which represents the ordering of the image in the storyboard
+async function generateImage(text, index, directoryPath) {
+    const fetch = require('node-fetch');
+    const fs = require('fs');
+
+    const engineId = 'stable-diffusion-v1-5';
+    const apiHost = process.env.API_HOST || 'https://api.stability.ai';
+    const apiKey = 'sk-7hCelYiEhiK4aHj12oAWQ65aOQXBJkK1CT3ld91HZoxp4aJx';
```

Comment:
I know it's bad to have api key hardcoded, but for simplicity's sake I kept it this way : p